### PR TITLE
Test refactor

### PR DESCRIPTION
**Pull request by jaadams5**
There are several files leftover after running tests.   Several of them are in data.  But there is one real file there as well.  I moved that file elsewhere, removed the CustomScriptTest that used it and then tried to replicate some of the script test as a FightRequest test.

### CODE REVIEW

**Comment by jaadams5** [comment]
This started out as a refactoring of a custom script test so the data directory could be empty after all tests had run.  The script used the test command to load and process.  The refactor does its own loading (from root/request) and then replicates the relevant actions from the test fight command.

Having moved the file, my hopes to automatically delete data were dashed because I don't have a hook that is invoked only after every test in every file has been run.

This, IMO, is still worth doing since the original script test predates some of the conventions we have adopted in testing.

File: test/net/sourceforge/kolmafia/request/FightRequestTest.java [hunk 1]
```diff
@@ -1077,6 +1078,26 @@ public void canDetectLilDoctorBagUses() {
...
   }
 
+  @Test
+  public void canDetectEvilometerChange() {
+    AdventureResult item = ItemPool.get(ItemPool.LIHC_EYE, 1);
+    assertEquals(0, item.getCount(KoLConstants.inventory));
+    var cleanups = new Cleanups(withFight(), withProperty("cyrptNicheEvilness", 50));
+    try (cleanups) {
+      assertThat("cyrptNicheEvilness", isSetTo(50));
+      String html = html("request/evilometer_fight.txt");
```
Comment (gausie) [inline test/net/sourceforge/kolmafia/request/FightRequestTest.java:1088]: Can you rename this file `test_fight_evilometer_single_loud_beep.html` or something

**Comment by midgleyc** [comment]
The original test did also serve as testing (and documentation, in the "code is documentation" style) of the "test" command, which this new version does not.

It might be nice to keep it as before, and test the outputs (with `Cleanups` to cleanup the new state). Like the other `CommandTest`s: `TestCommandTest`?

**Comment by jaadams5** [comment]
My core issue is just a file in data.  This addresses that.  But I could also make a special version of "test load" that was not forced to use data and then move this to a new TestCommandTest.  The CustomScriptTest could also be extended with a beforeAll and afterAll that copied files (currently only this one) into data before tests started and deleted them after.

I'm happy of the file is not in data when no tests are running.  One of the alternatives above might even be better because it creates some test infrastructure.

I'll mark this as draft while I decide whether to fix this approach or abandon it and use another.  I am open to suggestions :-)

File: test/net/sourceforge/kolmafia/request/FightRequestTest.java [hunk 2]
```diff
@@ -1077,6 +1078,26 @@ public void canDetectLilDoctorBagUses() {
     }
   }
 
+  @Test
+  public void canDetectEvilometerChange() {
+    AdventureResult item = ItemPool.get(ItemPool.LIHC_EYE, 1);
+    assertEquals(0, item.getCount(KoLConstants.inventory));
```
[the remark this answers is not in the document]
Comment (Veracity0) [reply test/net/sourceforge/kolmafia/request/FightRequestTest.java:1084]: Yes, I do that all the time when running a request will add something to inventory.

**Comment by jaadams5** [comment]
Closing.  Handled in 1256

**Pull request closed without merging**